Sources/Utility/Git.swift

```diff
@@ -57,7 +57,6 @@ public class Git {
         return knownVersions
     }
 
-    public class var tool: String {
-        return "git"
-    }
+    /// A shell command to run for Git. Can be either a name or a path.
+    public static var tool: String = "git"
```

Comment:
I am not sure how does this help? We expect to find git in PATH. If someone wants to override that and use a different git, they can do so by modifying their PATH. We do this for some of our tests e.g. https://github.com/apple/swift-package-manager/blob/master/Tests/FunctionalTests/MiscellaneousTests.swift#L424